### ISSUE TITLE
Custom spans don't show up in tracez by default

### ISSUE THREAD

**Issue by ramonza** [census-instrumentation/opencensus-go#438]
In order to show up in tracez, you need to set RecordEvents=true in StartOptions. This is not intuitive and isn't documented anywhere. We should either set RecordEvents=true everywhere or else document this behavior.

**Comment by ionutvilie** [comment]
continuing from: https://groups.google.com/forum/#!topic/census-developers/yryN1h-rdFs

first i can understand the behavior, i vote for documentation, next:

```golang
// import (  cntx "context" )
// cntx := cntx.Background()

// cntx, span := trace.StartSpanWithOptions(cntx, "foo",
// 	trace.StartOptions{RecordEvents: true})

cntx, span := trace.StartSpanWithOptions(cntx, "foo",
	trace.StartOptions{RecordEvents: true,
	RegisterNameForLocalSpanStore: true})

// cntx, span := trace.StartSpanWithOptions(cntx, "foo",
// 	trace.StartOptions{RecordEvents: true,
// 		Sampler: trace.AlwaysSample(),
// 		RegisterNameForLocalSpanStore: true})
```
Both `RecordEvents: true` and `RegisterNameForLocalSpanStore: true` must be set in order to work (tested). 

because of the tips below the working version for the tracez handler in iris-go
```golang

package main

import (
	cntx "context"
	"log"

	"github.com/kataras/iris"

	"go.opencensus.io/trace"
	"go.opencensus.io/zpages"

	"github.com/kataras/iris/middleware/logger"
	"github.com/kataras/iris/middleware/recover"
)

func bar(cntx cntx.Context) {
	cntx, span := trace.StartSpan(cntx, "/bar")
	defer span.End()
	// Do bar...
}

type exporter struct{}

func (e *exporter) ExportSpan(sd *trace.SpanData) {
	log.Println(sd)
}

func main() {

	app := iris.New()
	app.Use(recover.New())
	app.Use(logger.New())

	app.Get("/tracez", iris.FromStd(zpages.TracezHandler))

	app.Handle("GET", "/", func(ctx iris.Context) {

		cntx := cntx.Background()
		// cntx, span := trace.StartSpanWithOptions(cntx, "foo",
		// 	trace.StartOptions{RecordEvents: true,
		// 		Sampler: trace.AlwaysSample(),
		// 		RegisterNameForLocalSpanStore: true})

                // set trace StartOptions in order for the tracez handler to work.
		cntx, span := trace.StartSpanWithOptions(cntx, "foo",
			trace.StartOptions{RecordEvents: true,
				RegisterNameForLocalSpanStore: true})
		bar(cntx)
		defer span.End()
		ctx.HTML("<h1>Welcome</h1>")
	})

	app.Get("/ping", func(ctx iris.Context) {
		ctx.WriteString("pong")
	})
	app.Run(iris.Addr(":9090"), iris.WithoutServerError(iris.ErrServerClosed))
}
```
Thank you for the support. :)

**Comment by ramonza** [comment]
We are trying to remove the local span store from the trace API and make it part of the z-pages. In service of this, I'm going to try to default these StartOptions to true if the z-pages are imported. See #452

**Comment by ramonza** [comment]
Fixed in #452
